src/constants/collections.ts

```diff
@@ -0,0 +1,15 @@
+import { SiteKey } from "../types/site"
+
+export const QUESTION_TEMPLATE_COLLECTION_ID = 42
+
+export const SANDBOXED_CUSTOM_ANALYTICS_COLLECTIONS: Record<SiteKey, number> = {
+  stitch: 45,
+  luminara: 46,
+  pug: 47,
+}
+
+export const SANDBOXED_USER_GENERATED_COLLECTIONS: Record<SiteKey, number> = {
```

Comment:
These defines the "User-Generated" sub-collections inside of the above collection, so we can clean-up user-generated content with a cronjob.

![CleanShot 2567-09-04 at 02 01 33@2x](https://github.com/user-attachments/assets/ab645c2e-b686-4019-a754-d7abdf4b974d)